tests/dummy/app/controllers/bars.js

```diff
@@ -22,7 +22,7 @@ export default Ember.Controller.extend({
   }),
 
   tableContent: Ember.computed(function() {
-    return _.range(100).map(function(index) {
+    return Array.apply(null, new Array(100)).map(function(_, index) {
```

Comment:
I didn't even realize that we were only using it to iterate.

At the very least,

```js
var content = new Array(100);
return content.map(
    ...
);
```

should be enough.